addon/transforms/timestamp.js

```diff
@@ -13,7 +13,7 @@ export default class Timestamp extends Transform {
   }
 
   serialize(deserialized) {
-    if (typeOf(deserialized) !== 'date') {
+    if (typeOf(deserialized) !== 'date' && typeOf(deserialized) !== 'null') {
```

Comment:
Should this be `null` instead of `'null'`? `'null'` would end up being a string, right?

Reply:
Nevermind, I didn't notice you're doing a `typeOf` here. Making it `deserialized !== null` should work as well and is more straightforward.